src/include/sof/drivers/mu.h

```diff
@@ -1,13 +1,72 @@
 /* SPDX-License-Identifier: BSD-3-Clause
  *
  * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ * Copyright 2020 NXP
  *
  * Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
+ * Author: Daniel Baluta <daniel.baluta@nxp.com>
  */
 
 #ifndef __SOF_DRIVERS_MU_H__
 #define __SOF_DRIVERS_MU_H__
 
-#include <platform/drivers/mu.h>
+#include <sof/bit.h>
+#include <stdint.h>
+
+/* Transmit Register */
+#define IMX_MU_xTRn(x)		(0x00 + 4 * (x))
```

Comment:
This file has the same problem as ssp and mn drivers (maybe more) that it has vendor-specific stuff in project-wide headers. Won't disapprove because of that cos we did the same with Intel drivers, just letting you know it needs refactor in near future. It should include `<platform/drivers...>` and have these registers defined there per-platform/vendor.

Reply:
@jajanusz @dbaluta we should mark this as a task for v1.6

Reply:
@jajanusz I also noticed this, but couldn't quickly come with a place where to move it. Will refactor it in the future.

Reply:
> @jajanusz @dbaluta we should mark this as a task for v1.6

Hi Liam, I marked this as a task for v1.6. It is around our internal release date too.

Reply:
@dbaluta ok, great !
@jajanusz I guess we are now good for Intel headers ?

Reply:
@lgirdwood We are not good, it has to be fixed for 1.6. We have f.e. SSP registers offsets for our platforms in ssp.h that is included for all platforms. Created issue for this problem: https://github.com/thesofproject/sof/issues/2311